py_neuromodulation/nm_analysis.py

```diff
@@ -23,21 +22,22 @@
 
 
 class Feature_Reader:
-
-    feature_dir: str
-    feature_list: list[str]
-    settings: dict
-    sidecar: dict
-    sfreq: int
-    line_noise: int
-    nm_channels: pd.DataFrame
-    feature_arr: pd.DataFrame
-    ch_names: list[str]
-    ch_names_ECOG: list[str]
-    decoder: nm_decode.Decoder = None
+    
+#     These are class variables, not instance variables
```

Comment:
That's great. Makes total sense. Back then I was not aware of the difference.. 
Could you delete the comments? Thanks!

Reply:
Sure.